WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabItemsStore.swift

```diff
@@ -56,8 +56,8 @@ class ReaderTabItemsStore: ItemsStore {
 extension ReaderTabItemsStore {
 
     /// Fetch request to extract reader menu topics from Core Data
-    private var topicsFetchRequest: NSFetchRequest<NSFetchRequestResult> {
-        let fetchRequest = NSFetchRequest<NSFetchRequestResult>(entityName: ReaderTopicsConstants.entityName)
+    private var topicsFetchRequest: NSFetchRequest<ReaderAbstractTopic> {
```

Comment:
By using a concrete type here, we can avoid the need to test for invalid conversions, and remove a lot of error handling code.